lib/encoder/include/Pololu37D.h

```diff
@@ -22,8 +24,20 @@ class Pololu37D final : public Encoder {
   bool reset() override;
 
  private:
-  GPIO::QEI m_QEI;
+  bool read();
+  float return_delta_time();
+
+  // tracks the previous angle in degrees reading to calculate speed
+  float m_previous_angle_deg;
+  // holds the current angle in degrees
+  float m_current_angle_deg;
+  // holds the measured speed in degrees per second
+  float m_speed_deg_per_second;
```

Comment:
Recommend renaming to `m_angular_velocity_deg_per_sec`

Reply:
done